data/authors/julian.mdx

```diff
@@ -0,0 +1,22 @@
+---
+name: Julian 
+nym: Dynamic 
+avatar: /static/images/volunteers/julian.jpg
+occupation: Volunteer
+company: OpenSats
+email: julian@opensats.org
+nostr: npub16h5u7amjy5cwdxhru05mmql5cr954a2cpa2klj3685l72qzxm7wsskwy84
+github: https://github.com/julian2000-code
+volunteer: true
+---
+
+Julian's life experiences taught him the value of self-sovereignty. When he
+found Bitcoin, it opened his eyes to how technology can be used to enhance
+privacy and freedom instead of controlling others. Since then, Julian enjoys
+testing different self-sovereign tools, understanding the technical details
+behind how they work, and implementing his favorite tools into his everyday
```

Comment:
change from "implementing his favorite tools _into_" to "implementing his favorite tools _in_ his everyday life"